src/plots/smith/layout_attributes.js

```diff
@@ -0,0 +1,235 @@
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    angle: {
+        valType: 'angle',
+        editType: 'plot',
+        description: [
+            'Sets the angle (in degrees) from which the radial axis is drawn.',
+            'Note that by default, radial axis line on the theta=0 line',
+            'corresponds to a line pointing right (like what mathematicians prefer).',
+            'Defaults to the first `polar.sector` angle.'
+        ].join(' ')
+    },
+
+    side: {
+        valType: 'enumerated',
+        // TODO add 'center' for `showline: false` radial axes
+        values: ['clockwise', 'counterclockwise'],
+        dflt: 'clockwise',
+        editType: 'plot',
+        description: [
+            'Determines on which side of radial axis line',
+            'the tick and tick labels appear.'
+        ].join(' ')
+    },
+
+
+    title: {
+        // radial title is not gui-editable at the moment,
+        // so it needs dflt: '', similar to carpet axes.
+        text: extendFlat({}, axesAttrs.title.text, {editType: 'plot', dflt: ''}),
+        font: extendFlat({}, axesAttrs.title.font, {editType: 'plot'}),
+
+        // TODO
+        // - might need a 'titleside' and even 'titledirection' down the road
+        // - what about standoff ??
+
+        editType: 'plot'
+    },
+
+    hoverformat: axesAttrs.hoverformat,
+
+    uirevision: {
+        valType: 'any',
+        editType: 'none',
+        description: [
+            'Controls persistence of user-driven changes in axis `range`,',
+            '`autorange`, `angle`, and `title` if in `editable: true` configuration.',
+            'Defaults to `polar<N>.uirevision`.'
+        ].join(' ')
+    },
+
+    editType: 'calc',
+
+    _deprecated: {
+        title: axesAttrs._deprecated.title,
+        titlefont: axesAttrs._deprecated.titlefont
+    }
+};
+
+extendFlat(
+    radialAxisAttrs,
+
+    // N.B. realaxis grid lines are circular,
+    // but realaxis lines are straight from circle center to outer bound
+    axisLineGridAttr,
+    axisTickAttrs
+);
+
+var angularAxisAttrs = {
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    direction: {
+        valType: 'enumerated',
+        values: ['counterclockwise', 'clockwise'],
+        dflt: 'counterclockwise',
+        editType: 'calc',
+        description: [
+            'Sets the direction corresponding to positive angles.'
+        ].join(' ')
+    },
+
+    rotation: {
+        valType: 'angle',
+        editType: 'calc',
+        description: [
+            'Sets that start position (in degrees) of the angular axis',
+            'By default, polar subplots with `direction` set to *counterclockwise*',
+            'get a `rotation` of *0*',
+            'which corresponds to due East (like what mathematicians prefer).',
+            'In turn, polar with `direction` set to *clockwise* get a rotation of *90*',
+            'which corresponds to due North (like on a compass),'
```

Comment:
polar > smith ?

Reply:
I fixed this in one of the previous commits (not sure which one).